request-comply-advantage/README.md

```diff
@@ -0,0 +1,424 @@
+# Request Comply Advantage Blacklist
+This README contains a brief description of [the example content](./index.js)
+
+## What exactly is Comply Advantage?
+Comply Advantage is a tech company that creates a solution detecting and reducing fraud losses. The Comply Advantage focus is to identify and reduce the risks associated with illegal financial activities.
```

Comment:
*that  creates solutions to detect and reduce fraud and associated losses.

Creo que fluye un poco mejor, técnicamente tienen varios productos.

Reply:
Tambien: *Comply Advantage's focus